### PR TITLE
Remove custom layout validation

### PR DESCRIPTION
**Pull request by DilshanSenarath**
### Purpose

When activating a custom layout from the branding design section, the system currently checks for the existence of a custom layout within the current tenant. However, to perform this validation accurately, a centralized location for delivering these layouts is necessary. Currently, these layouts are stored in the authentication portal and recovery portal, inaccessible to the console due to cross-app restrictions. Consequently, validation will be disabled. There is an ongoing issue being tracked to implement centralized support for resources (https://github.com/wso2/product-is/issues/16706). This requirement can be taken into account as part of the solution for that issue.

### Issues

- N/A

### Checklist
- [ ] e2e cypress tests locally verified.
- [ ] Manual test round performed and verified.
- [ ] UX/UI review done on the final implementation.
- [ ] Documentation provided. (Add links if there are any)
- [ ] Unit tests provided. (Add links if there are any)
- [ ] Integration tests provided. (Add links if there are any)

### Security checks
- [ ] Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines?
- [ ] Ran FindSecurityBugs plugin and verified report?
- [ ] Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets?

### CODE REVIEW

**Comment by pavinduLakshan** [comment]
Shall we update the issue description with the corresponding product-is issue for the changes in this PR if there is any?

**Comment by DilshanSenarath** [comment]
> Shall we update the issue description with the corresponding product-is issue for the changes in this PR if there is any?

There is no specific issue for this. Since this is a limitation, we are tracking to improve a solution for this in https://github.com/wso2/product-is/issues/16706.

File: apps/console/src/extensions/configs/models/common.ts [hunk 1]
```diff
@@ -24,6 +24,7 @@ export interface CommonConfig {
         enableQuerySearch: boolean;
     };
     blockLoopBackCalls: boolean;
+    checkCustomLayoutExistanceBeforeEnabling: boolean;
```
Comment (pavinduLakshan) [inline apps/console/src/extensions/configs/models/common.ts:27]: Let's update the cloud extension configs as well.

File: apps/console/src/features/branding/components/design/design-form.tsx [hunk 2]
```diff
@@ -192,7 +192,7 @@ export const DesignForm: FunctionComponent<DesignFormPropsInterface> = forwardRe
     const {
         data: customLayoutBlob,
         isLoading: customLayoutLoading
-    } = useLayout(PredefinedLayouts.CUSTOM, tenantDomain);
+    } = useLayout(PredefinedLayouts.CUSTOM, tenantDomain, commonConfig.checkCustomLayoutExistanceBeforeEnabling);
```
Comment (pavinduLakshan) [inline apps/console/src/features/branding/components/design/design-form.tsx:195]:
```suggestion
    } = useLayout(PredefinedLayouts.CUSTOM, tenantDomain, commonConfig?checkCustomLayoutExistanceBeforeEnabling);
```

File: apps/console/src/features/branding/components/design/design-form.tsx [hunk 3]
```diff
@@ -335,7 +335,8 @@ export const DesignForm: FunctionComponent<DesignFormPropsInterface> = forwardRe
         }
 
         // Avoid activating the custom layout when there is no deployed custom layout.
-        if (_values.layout.activeLayout === PredefinedLayouts.CUSTOM) {
+        if (commonConfig.checkCustomLayoutExistanceBeforeEnabling
```
Comment (pavinduLakshan) [inline apps/console/src/features/branding/components/design/design-form.tsx:338]:
```suggestion
        if (commonConfig?.checkCustomLayoutExistanceBeforeEnabling
```

File: apps/console/src/features/branding/components/design/design-form.tsx [hunk 4]
```diff
@@ -335,7 +335,8 @@ export const DesignForm: FunctionComponent<DesignFormPropsInterface> = forwardRe
...
+                && _values.layout.activeLayout === PredefinedLayouts.CUSTOM) {
```
Comment (pavinduLakshan) [inline apps/console/src/features/branding/components/design/design-form.tsx:339]:
```suggestion
                && _values?.layout?.activeLayout === PredefinedLayouts.CUSTOM) {
```

File: apps/console/src/extensions/configs/models/common.ts (hunk 1 above)
Comment (DilshanSenarath) [reply apps/console/src/extensions/configs/models/common.ts:27]: Sure, once this is released will update the cloud extensions with the bump for the new version.

File: apps/console/src/features/branding/components/design/design-form.tsx (hunk 2 above)
Comment (DilshanSenarath) [reply apps/console/src/features/branding/components/design/design-form.tsx]: Fixed with https://github.com/wso2/identity-apps/pull/5078/commits/f05d41c38d4c377d25330f7963fad3f66e093769

File: apps/console/src/features/branding/components/design/design-form.tsx (hunk 3 above)
Comment (DilshanSenarath) [reply apps/console/src/features/branding/components/design/design-form.tsx]: Fixed with https://github.com/wso2/identity-apps/pull/5078/commits/cc2f8ed3b6d31ba5374f5e32093a350c3518f9ec

File: apps/console/src/features/branding/components/design/design-form.tsx (hunk 4 above)
Comment (DilshanSenarath) [reply apps/console/src/features/branding/components/design/design-form.tsx]: Fixed with https://github.com/wso2/identity-apps/pull/5078/commits/cc2f8ed3b6d31ba5374f5e32093a350c3518f9ec

**Review by pavinduLakshan — APPROVED** [review]

**Pull request merged**